### DISCUSSION TITLE
Permission set is generated with incorrect name (force-app), is it a known issue?

### DISCUSSION THREAD

**Discussion by tamastom** [scolladon/sfdx-git-delta#864] [General]
<img width="941" alt="image" src="https://github.com/scolladon/sfdx-git-delta/assets/8284005/ec840348-7c3d-46b4-8178-563b28ceb838">

I reverted to version 5.39.1 but still the same issue.

**Comment by scolladon** [comment]
Hi @tamastom !

It seems to be similar to an [old issue](https://github.com/scolladon/sfdx-git-delta/issues/834)

Could you create an issue and follow the issue template as much as possible please ?
I need more information to reproduce and isolate the issue so I can troubleshoot and see what is going on.
